src/Analyzers/CSharp/Analyzers/UseImplicitOrExplicitType/CSharpUseExplicitTypeDiagnosticAnalyzer.cs

```diff
@@ -7,14 +7,20 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Diagnostics.TypeStyle
 {
+#if CODE_STYLE
```

Comment:
These #if statements should go away once @tmat's change https://github.com/dotnet/roslyn/pull/34003 goes in and we can use shared resource files for all analyzer and code fix resources that are shared between Workspaces, Features and CodeStyle layer.